### PR TITLE
Add Gofmt action inside Dynamic Analysis section

### PR DESCRIPTION
**Pull request by Jerome1337**
[Marketplace link](https://github.com/marketplace/actions/gofmter)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -277,6 +277,10 @@ Set up your GitHub Actions workflow with a specific version of your programming
 - [Publishing code coverage to CodeClimate](https://github.com/paambaati/codeclimate-action)
 - [Update repository go report card](https://github.com/creekorful/goreportcard-action)
 
+#### Code Quality
+
+- [Run Gofmt to check Golang code formatting](https://github.com/Jerome1337/gofmt-action)
```
Comment (jessicalostinspace) [inline README.md:282]: Again, this changes the code as opposed to static analysis.

Comment (jessicalostinspace) [reply README.md:282]: Also, please make sure all of your titles are Title Cased: https://github.com/sdras/awesome-actions/blob/master/contributing.md

Comment (Jerome1337) [reply README.md:292]: @jessicalostinspace I've made some changes

Comment (sdras) [reply README.md:292]: This still isn't title cased

**Pull request merged**